src/components/Preview/Preview.js

```diff
@@ -14,5 +14,11 @@ export default function Preview() {
   //   <p>This is some blog text. </p>
   // </div>
 
-  return <></>;
+  return (
+    <div className="preview" style={{ fontFamily: `${font}`, textAlign: `${alignment}` }}>
```

Comment:
You are really close here, but the text isn't changing correctly with the way you have it set here! Take a look at the preview.css file and see if you can spot the problem. Let me know if you want to talk it over.